package-lock.json

```diff
@@ -0,0 +1,11656 @@
+{
```

Comment:
Please remove this file.

Reply:
Sir, I couldn't find "@@ -0,0 +1,11656 @@" this in my package-lock.json file. Maybe these are added while merging?